src/screens/Home/Home.tsx

```diff
@@ -37,6 +38,70 @@ interface HomeProps {
 
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
+  useEffect(() => {
+    proccessData();
+  }, []);
+
+  const [outStandingPayments, setOutstandingPayments] = useState<number>();
+  const [toCharge, setToCharges] = useState<number>();
+  const [customerDetails, setCustomerDetails] = useState<number>();
+  const [unpaidReports, setUnpaidReports] = useState<number>();
+
+
+
+  const proccessData = () => {
+    calculateCustomerData(currentSite);
+    calculateUnpaidReports(currentSite)
+  }
+
+  const calculateCustomerData = (ss: SiteRecord) => {
+    // customers to charge:  haven't charged yet/ no meterreading
+    // outstanding payments: done the meter reading / charged, haven't paid yet
+    let toCharge = 0;
+    let outstanding = 0;
+    //get day
+    const today = new Date();
+    const currMonth = today.getUTCMonth();
+    let customers = ss.customers;
+    for (let i = 0; i < customers.length; i++) {
+      let curr = customers[i]
+      //depends if the meter readings list is sorted with earliest => latest
+      let latestMeterReading = curr.meterReadings[curr.meterReadings.length - 1];
+      if (latestMeterReading != null) {
+        const dateTime = Date.parse(latestMeterReading.date);
+        let readingDate = new Date(dateTime);
+        if (readingDate.getUTCMonth() < currMonth) {
```

Comment:
Write a util function in the moment utils file so we have a standardized way of date comparison. The function could pass in the string dates we get from Airtable and do proper comparisons